### DISCUSSION TITLE
Included webserver

### DISCUSSION THREAD

**Discussion by Bluscream** [Rem0o/FanControl.Releases#789] [Ideas]
Since this is basically just a local WebUi, it would be nice if the interface was accessible from the LAN via a internal webserver

**Comment by Rem0o** [comment]
It's not a "local WebUi", it's a desktop WPF app. Making a web-server out of it is a whole new project.

**Reply by Bluscream** [reply]
It really is WPF? Damn i thought wpf was wayy more responsive than that xd

**Reply by Rem0o** [reply]
More responsive? How?
